charts/series.go

```diff
@@ -325,7 +325,7 @@ func WithLineChartOpts(opt opts.LineChart) SeriesOpts {
 	}
 }
 
-// WithLineChartOpts sets the LineChart option.
+// WithKlineChartOpts sets the LineChart option.
 func WithKlineChartOpts(opt opts.KlineChart) SeriesOpts {
```

Comment:
For the typo fix,  it seems strange for me here since the crude `LineChart option` is still exist.